### PR TITLE
November address attributions

### PR DESCRIPTION
**Pull request by jeffdefacto**
### Docs Preview:

[Docs preview for this PR.](https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/129)

View all staged runs:
https://github.com/OvertureMaps/docs/actions/workflows/publish-pr-to-staging.yml

### CODE REVIEW

File: docs/attribution.mdx [hunk 1]
```diff
@@ -16,62 +16,102 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
...
+- Contains data from the [Danish Agency for Data Supply and Infrastructure](https://eng.sdfi.dk/). [Terms of use for free geographic data](https://dataforsyningen.dk/asset/PDF/rettigheder_vilkaar/Vilk%C3%A5r%20for%20brug%20af%20frie%20geografiske%20data.pdf). Distributed by OpenAddresses. Accessed: 2024-11-10.
 #### Estonia
-- Data courtesy of [Estonian Land Board (MAA-AMET)](https://geoportaal.maaamet.ee/eng/). Available under [CC0](https://creativecommons.org/public-domain/cc0/). Distributed by OpenAddresses. Accessed: 2024-10-13.
+- Data courtesy of [Estonian Land Board (MAA-AMET)](https://geoportaal.maaamet.ee/eng/). Available under [CC0](https://creativecommons.org/public-domain/cc0/). Distributed by OpenAddresses. Accessed: 2024-11-10.
 #### Finland
-- © Väestörekisterikeskus. Available under CC BY 4.0. Distributed by OpenAddresses. Accessed: 2024-10-13.
+- © Väestörekisterikeskus. Available under CC BY 4.0. Distributed by OpenAddresses. Accessed: 2024-11-10.
```
Comment (stepps00) [inline docs/attribution.mdx:41]: Looking at the OA source, we might be able to link to https://www.avoindata.fi/data/fi/dataset/rakennusten-osoitetiedot-koko-suomi here

File: docs/attribution.mdx [hunk 2]
```diff
@@ -16,62 +16,102 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
...
 #### France
-- Data from [The national address site of the French Republic](https://adresse.data.gouv.fr/). Available under [LICENCE OUVERTE / OPEN LICENCE V2 - Etalab 2.0](https://etalab.gouv.fr/). Distributed by OpenAddresses. Accessed: 2024-10-13.
+- Data from [The national address site of the French Republic](https://adresse.data.gouv.fr/). Available under [LICENCE OUVERTE / OPEN LICENCE V2 - Etalab 2.0](https://etalab.gouv.fr/). Distributed by OpenAddresses. Accessed: 2024-11-10.
```
Comment (stepps00) [inline docs/attribution.mdx:43]: License page is now located at: https://www.etalab.gouv.fr/wp-content/uploads/2017/04/ETALAB-Licence-Ouverte-v2.0.pdf

File: docs/attribution.mdx [hunk 3]
```diff
@@ -16,62 +16,102 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
...
+- Berlin data from [Geoportal Berlin](https://daten.berlin.de/tags/amtlichen-hauskoordinaten). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2024-11-10.
+- Brandenburg data from [GeoBasis-DE/LGB](https://geobasis-bb.de/lgb/de/geodaten/liegenschaftskataster/georeferenzierte-adresse/). Available under [DL-DE->BY-2.0](https://www.govdata.de/dl-de/by-2-0). Distributed by OpenAddresses. Accessed: 2024-11-10.
+- Bremen data from [Landesamt GeoInformation Bremen](https://www.geo.bremen.de/produkte/katasterprodukte/auszuege-aus-dem-liegenschaftskataster-12272). Available under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/). Distributed by OpenAddresses. Accessed: 2024-11-10. 
+- City of Rostock data from [Hanse- und Universitätsstadt Rostock │](http://www.opendata-hro.de/). Available under [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/). Distributed by OpenAddresses. Accessed: 2024-11-10.
```
Comment (stepps00) [inline docs/attribution.mdx:49]: Extra ` |` in the source name?

File: docs/attribution.mdx [hunk 4]
```diff
@@ -16,62 +16,102 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
 ### Addresses
 **License for theme:** The addresses data comes from a variety of sources. All carry permissive open licenses. Some have special terms or require attribution. These terms are outlined below.
 #### Australia
-- G-NAF © [Geoscape Australia](https://geoscape.com.au/) licensed by the Commonwealth of Australia. Available under the [Open Geo-coded National Address File End User Licence Agreement](https://geoscape.com.au/wp-content/uploads/2022/08/EULA-G-NAF-Core-1.pdf). Distributed by OpenAddresses. Accessed: 2024-10-13.
+- G-NAF © [Geoscape Australia](https://geoscape.com.au/) licensed by the Commonwealth of Australia. Available under the [Open Geo-coded National Address File End User Licence Agreement](https://geoscape.com.au/wp-content/uploads/2022/08/EULA-G-NAF-Core-1.pdf). Distributed by OpenAddresses. Accessed: 2024-11-10.
```
Comment (stepps00) [inline docs/attribution.mdx:19]: Looks like this license page is now at: https://geoscape.com.au/wp-content/uploads/2024/08/EULA-G-NAF-Core-1.pdf

**Review by stepps00 — APPROVED** [review]
A couple nits about links and formatting, otherwise looking great

**Review by danabauer — CHANGES_REQUESTED** [review]
addresses is getting to be a beast! let's make the attribution list for each theme a collapsible section:
```
<details>
    <summary>
    **Addresses**
    </summary>
attribution list here
</details>
```

**Comment by jeffdefacto** [comment]
Definitely cleaner now

**Review by danabauer — APPROVED** [review]

**Pull request merged**